project/settings.sc

```diff
@@ -836,8 +836,9 @@ trait ScalaCliModule extends ScalaModule {
   }
 }
 
-def workspaceDirName = ".scala-build"
-def projectFileName  = "project.scala"
+def workspaceDirName      = ".scala-build"
+def projectFileName       = "project.scala"
+def jvmPropertiesFileName = ".scala-jvmopts"
```

Comment:
why `scala-jvmopts` ? That's not a standard name, so might be ahrd to discover.

Reply:
I'd vote for the name `.scala-jvmopts` to prevent potential collision with other tools that use `.jvm-opts`. Our native launcher restricts us from supporting every Java option added in `.jvmopts` and can only pass Java properties during Scala CLI execution.

The possibility of Scala CLI printing warning, unable to load some options like `-Xmx6g` from `.jvm-opts` in `scala-cli` launcher, can lead to confusion. Hence, I suggest using `.scala-jvmopts` name.

Reply:
> The possibility of Scala CLI printing warning, unable to load some options like -Xmx6g from .jvm-opts in scala-cli launcher, can lead to confusion. Hence, I suggest using .scala-jvmopts name.

I would just mention that this is a native image and it doesn't support those options. Besides I am not sure this would be a common scenario. `.scala-jvmopts` would be very hard to discover and would require the user to dig into documentation, which we should avoid.